config/config.yaml

```diff
@@ -0,0 +1,90 @@
+# netbox device attributes that should be used as filters for devices query.
+# these translate directly into the return fields from the netbox json API.
+# If no filter is specified, by default only devices with Active (1) status are pulled.
+filters:
+  status: ["active", "staged", "offline"]
+#  manufacturer: ['juniper']
+#  custom_fields.field1: "foo"
+
+# list of tags to filter devices in Netbox. Only devices that have this tag will be 
+# synchronized to NetSpyGlass. Default is None, which causes all devices present in Netbox to be synchronized.
+whitelist:
+#  - tag1
+#  - tag2
+
+# list of tags to blacklist from netbox. Devices that have this tag will not be synchronized to NetSpyGlass.
+# Blacklist filter is applied after the whitelist (if any). Default is None, which turns this off and causes all
+# devices that pass whitelist will be synchronized
+blacklist:
+#  - tagq
+
+# list of devices in nsg excluding from synchronization 
+nsg_blacklist:
```

Comment:
show one or two examples how to use this config paraeter (in the comment)

and consider more descriptive name, e.g. nsg_device_filter or something like that